initial_bot/mentor_bot_test.py

```diff
@@ -19,85 +61,97 @@ def parse_arguments():
                         default="mentorbot_test")
     return parser.parse_args()
 
+intents = discord.Intents.all()
+bot = commands.Bot(command_prefix = '!', intents = intents, case_insensitive=True)
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+stream_handler = logging.StreamHandler()
+stream_format = logging.Formatter('%(levelname)s - %(message)s')
+stream_handler.setFormatter(stream_format)
+
+logger.addHandler(stream_handler)
+
+@bot.event
+async def on_ready():
+    logger.info(f'{bot.user.name} has connected to Discord!')
+
+@bot.event
+async def on_message(message):
+    '''        
+    private mentoring rooms will have some identifier in their names, so the bot can decide whether or not to save the message.
+    For now I am tentatively using "shlomi" as the identifier :-)
+    
+    '''
+    private_mentor_channels_mark = '_mentors_'
+    channel = message.channel
+    author = message.author
+    if (not author.bot) and (private_mentor_channels_mark in channel.name):
+        with open(f"initial_bot/{channel.name}_messages.txt", 'a+') as f:
```

Comment:
Lets save the messages to the database, not to a file collection, what will we do with a collection of files?